build.rs

```diff
@@ -15,3 +21,29 @@ fn manage_docs() {
 
 #[cfg(not(any(feature = "embed-lgpl-docs", feature = "purge-lgpl-docs")))]
 fn manage_docs() {}
+
+#[cfg(feature = "test")]
+fn generate_tests() {
+    let output = Command::new("glib-compile-schemas")
```

Comment:
Curious to see if this works on Windows like this :)

Reply:
Uhh..same, but I don't have a windows maachine at my disposal

Reply:
There's Windows CI :)

Reply:
Oh, missed that :P